.eslintrc.yml

```diff
@@ -43,13 +47,13 @@ settings:
     version: 16.3.2
 
 overrides:
-  - files: '*.mdx'
+  - files: '*.{md,mdx}'
     extends:
       - plugin:mdx/recommended
       - plugin:mdx/overrides
     rules:
       import/no-extraneous-dependencies: 0
-      prettier/prettier: 0
+      prettier/prettier: 2
```

Comment:
it would be good to use `off` and `error` for consistency with the rest of the config and clarity

Reply:
Done!